test/integration/provider/rti.go

```diff
@@ -0,0 +1,139 @@
+package provider
+
+import (
+	"fmt"
+
+	gcp "github.com/gardener/machine-controller-manager-provider-gcp/pkg/gcp"
+	v1alpha1 "github.com/gardener/machine-controller-manager/pkg/apis/machine/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type ResourcesTrackerImpl struct {
+	InitialVolumes   []string
+	InitialInstances []string
+	InitialMachines  []string
+	MachineClass     *v1alpha1.MachineClass
+	SecretData       map[string][]byte
+	ClusterName      string
+}
+
+func (r *ResourcesTrackerImpl) InitializeResourcesTracker(machineClass *v1alpha1.MachineClass, secretData map[string][]byte, clusterName string) error {
+
+	clusterTag := "kubernetes-io-cluster-" + clusterName
+	clusterTagValue := "1"
+	r.MachineClass = machineClass
+	r.SecretData = secretData
+	r.ClusterName = clusterName
+
+	ms := gcp.NewGCPPlugin(&gcp.PluginSPIImpl{})
+	ctx, svc, err := ms.SPI.NewComputeService(&corev1.Secret{Data: secretData})
+	if err != nil {
+		return err
+	}
+
+	instances, err := GetInstancesWithTag(ctx, svc, "mcm-integration-test-true", machineClass, secretData)
```

Comment:
`mcm-integration-test-true` if these tags are commonly being used for this IT test purposes then it will be a good idea to make them global variables. I have to do similar changes with Azure as well.

Reply:
makes sense , I will make it a global variable in `rti.go`